autopush/router/webpush.py

```diff
@@ -60,7 +60,8 @@ def _send_notification(self, uaid, node_id, notification):
         """
         payload = {"channelID": notification.channel_id,
                    "version": notification.version,
-                   "ttl": notification.ttl+int(time.time()),
+                   "ttl": notification.ttl,
+                   "timestamp": int(time.time()),
```

Comment:
If we're not doing vector clocks or clock sync, I wonder if we really need to have a timestamp here? TTL could be set as the storage time+TTL on the machine that will reap.